core-plugin/testSrc/com/google/gct/idea/appengine/validation/EndpointTestBase.java

```diff
@@ -75,4 +86,24 @@ public void run() {
       }
     }
   }
+
+  private void removeEndpointsSdkFromProject() {
+    ApplicationManager.getApplication().runWriteAction(new Runnable() {
+      @Override
+      public void run() {
+        final LibraryTable table =
```

Comment:
There's no reason, I must have just copied it from the "addEndpointsSdk..." code above, will change it back to be consistent.